### PR TITLE
Add blueprint@2.1.5 w/ npm autoupdate

### LINKED ISSUE #9587: [Request] Add blueprint
**Library name:** blueprint
**Git repository url:** https://github.com/avinoamr/blueprint.js
**npm package url(optional):** https://www.npmjs.com/package/blueprint
**License(s):** MIT

### PR DESCRIPTION
**Pull request by extend1994**
PR for #9587  

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/avinoamr/blueprint.js
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/blueprint
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 4
   - Count of watchers: 1
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 21
   - Downloads in the last week: 175
   - Downloads in the last month: 788

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9587 , cc @avinoamr

### CODE REVIEW

File: ajax/libs/blueprint/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/avinoamr/blueprint.js.git"
+  },
+  "npmName": "blueprint",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```
Comment (pvnr0082t) [inline ajax/libs/blueprint/package.json]: @extend1994 Let's use `"blueprint*"` here

**Comment by extend1994** [comment]
@pvnr0082t please review this pull request, thank you!

**Comment by extend1994** [comment]
@pvnr0082t I have done minify again, please help me review it again, thank you.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@extend1994 I think you should tag me once the CI test finished.

**Pull request merged**